web/src/popovers.js

```diff
@@ -987,61 +986,11 @@ export function register_click_handlers() {
         current_user_sidebar_popover = $target.data("popover");
     });
 
-    $("body").on("click", ".remind.custom", (e) => {
-        $(e.currentTarget)[0]._flatpickr.toggle();
-        e.stopPropagation();
-        e.preventDefault();
-    });
-
-    function reminder_click_handler(datestr, e) {
-        const message_id = $(".remind.custom").data("message-id");
-        reminder.do_set_reminder_for_message(message_id, datestr);
-        hide_all();
-        e.stopPropagation();
-        e.preventDefault();
-    }
-
-    $("body").on("click", ".remind.in_20m", (e) => {
-        const datestr = formatISO(add(new Date(), {minutes: 20}));
-        reminder_click_handler(datestr, e);
-    });
-
-    $("body").on("click", ".remind.in_1h", (e) => {
-        const datestr = formatISO(add(new Date(), {hours: 1}));
-        reminder_click_handler(datestr, e);
-    });
-
-    $("body").on("click", ".remind.in_3h", (e) => {
-        const datestr = formatISO(add(new Date(), {hours: 3}));
-        reminder_click_handler(datestr, e);
-    });
-
-    $("body").on("click", ".remind.tomo", (e) => {
-        const datestr = formatISO(
-            set(add(new Date(), {days: 1}), {hours: 9, minutes: 0, seconds: 0}),
-        );
-        reminder_click_handler(datestr, e);
-    });
-
-    $("body").on("click", ".remind.nxtw", (e) => {
-        const datestr = formatISO(
-            set(add(new Date(), {weeks: 1}), {hours: 9, minutes: 0, seconds: 0}),
-        );
-        reminder_click_handler(datestr, e);
-    });
-
     $("body").on("click", ".flatpickr-calendar", (e) => {
         e.stopPropagation();
         e.preventDefault();
     });
```

Comment:
Added this back in because removing it makes the calendar close the compose box when an arrow for the time or month is clicked.

Reply:
OK -- might be worth discussing in #frontend if we want to do something else long term.

Reply:
CZO topic: https://chat.zulip.org/#narrow/stream/6-frontend/topic/flatpickr.20click.20handler/near/1560765